WordPress/src/main/res/layout/my_site_fragment.xml

```diff
@@ -2,634 +2,677 @@
+                        android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
-                        android:layout_width="match_parent"
-                        android:minHeight="@dimen/min_touch_target_sz"
-                        android:orientation="horizontal"
-                        android:paddingStart="@dimen/my_site_list_row_padding_left_switch"
-                        tools:ignore="RtlSymmetry,UnusedAttribute" >
+                        android:layout_gravity="center"
+                        android:ellipsize="end"
+                        android:fontFamily="sans-serif-medium"
+                        android:gravity="center_vertical"
+                        android:maxLines="2"
+                        android:textAlignment="center"
+                        android:textColor="@color/neutral_70"
+                        android:textSize="@dimen/text_sz_extra_large"
+                        tools:text="Example"/>
 
-                        <ImageView
-                            android:importantForAccessibility="no"
-                            android:layout_height="wrap_content"
-                            android:layout_gravity="center_vertical"
-                            android:src="@drawable/ic_arrow_left_white_24dp"
-                            style="@style/MySiteListRowIcon" >
-                        </ImageView>
+                    <org.wordpress.android.widgets.WPTextView
+                        android:id="@+id/my_site_subtitle_label"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center"
+                        android:layout_marginTop="@dimen/margin_small"
+                        android:ellipsize="end"
+                        android:gravity="center_vertical"
+                        android:maxLines="1"
+                        android:textColor="@color/neutral"
+                        android:textSize="@dimen/text_sz_medium"
+                        tools:text="example.wordpress.com"/>
+                </LinearLayout>
+
+                <org.wordpress.android.widgets.WPTextView
+                    android:id="@+id/switch_site"
+                    android:layout_width="wrap_content"
+                    android:layout_height="36dp"
+                    android:layout_gravity="center"
+                    android:layout_marginBottom="@dimen/margin_extra_extra_medium_large"
+                    android:layout_marginTop="@dimen/margin_medium"
+                    android:background="?attr/selectableItemBackground"
+                    android:ellipsize="end"
+                    android:fontFamily="sans-serif-medium"
+                    android:gravity="center_vertical"
+                    android:maxLines="1"
+                    android:text="@string/my_site_btn_switch_site"
```

Comment:
```suggestion
                    android:text="@string/my_site_btn_switch_site"
                    android:padding="@dimen/margin_medium"
```

Can we add the `android:padding="@dimen/margin_medium"` attribute to the `switch_site` view?  That will add left/start and right/end padding to the text button with the default text size as well as ensure the top and bottom padding exists for large text sizes.

Reply:
Huh. I thought I have added padding already 😕